app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def compare_clean_mark_and_power(self, car: Car) -> bool:
+        return car.clean_mark < self.clean_power
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0.0
+
+        for car in cars:
+            if self.compare_clean_mark_and_power(car):
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(car.comfort_class
+                     * (self.clean_power
+                        - car.clean_mark)
+                     * self.average_rating
+                     / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.compare_clean_mark_and_power(car):
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate: int) -> None:
+        self.average_rating = round((self.count_of_ratings
+                                    * self.average_rating
+                                    + rate)
+                                    / (self.count_of_ratings
+                                    + 1), 1)
```

Comment:
```suggestion
        self.average_rating = round(
                                    (self.count_of_ratings * self.average_rating + rate)
                                    / (self.count_of_ratings + 1), 1
                                    )
```

Reply:
I get this error from flake8 if I try to paste your self.average_rating method into the editor: 
"continuation line over-indented for hanging indentFlake8(E126)". 
I fixed the formatting so I don't get this error.